pallets/asset-manager/src/lib.rs

```diff
@@ -394,6 +396,79 @@ pub mod pallet {
 		pub fn account_id() -> T::AccountId {
 			T::PalletId::get().into_account()
 		}
+
+		pub fn set_genesis_configuration() -> Weight {
+			let mut weight: Weight = T::DbWeight::get().reads(1);
+			if have_storage_value(Self::name().as_bytes(), b"NextAssetId", &[]) {
+				weight = weight.saturating_add(T::DbWeight::get().reads(1));
```

Comment:
since you start with 1R in L401, I think this line shouldn't be here

Reply:
indeed